### ISSUE TITLE
Kubernetes Firewall rules error in Guest Network

### ISSUE THREAD

**Issue by Arceoavs** [apache/cloudstack#6929]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
Kubernetes Service
~~~

##### CLOUDSTACK VERSION

~~~
4.17.1.0
~~~

##### CONFIGURATION

- Kubernetes Cluster v1.23 deployed via the Cloudstack kubernetes service in a guest network. Non-HA, Cluster running successfully. 
- Advanced Network with Security Groups.

##### OS / ENVIRONMENT

N/A

##### SUMMARY

When setting firewall rules via the Cloudstack UI for the kubernetes cluster, this error message appears: 
```
Request failed. (431)
Unable to execute API command createfirewallrule due to missing parameter ipaddressid
```

##### STEPS TO REPRODUCE

~~~
Create a kubernetes cluster via the coudstack UI using a guest network
When the cluster is runnning, navigate to the cluster page > Firewall in the UI
Create a firewall rule (e.g. Source CIDR: 0.0.0.0/0, Protocol: TCP, Start port: 1234, End port: 1235)
Add the rule
~~~

##### EXPECTED RESULTS

~~~
The rule is added and displayed on the page.
~~~

##### ACTUAL RESULTS

~~~
Request failed. (431)
Unable to execute API command createfirewallrule due to missing parameter ipaddressid
~~~

**Comment by rohityadavcloud** [comment]
cc @davidjumani @shwstppr

**Comment by weizhouapache** [comment]
fixed by #7076
